### ISSUE TITLE
Be able to resize text in the Language Generation editor or choose font

### ISSUE THREAD

**Issue by JonathanFingold** [microsoft/BotFramework-Composer#5766]
In the properties pane on my machine the default font for the **Language Generation** editor appears to be using a 16 or 18 point font, which makes it hard to use.

When trying to find a way to zoom out or change the font, I got a very confusing context menu, which is perhaps a bug:

![image](https://user-images.githubusercontent.com/35047199/107689131-52b0aa00-6c5d-11eb-8ff5-602e1caad82f.png)

**Comment by JonathanFingold** [comment]
After being annoyed by this for a bit, I finally figured out how to use the context menu.
1. Select the **Command Palette**, which brings up an equally confusing menu:
    ![image](https://user-images.githubusercontent.com/35047199/107701331-9f03e600-6c6d-11eb-8dc9-7e1342029039.png)
2. Then search on "zoom", as scrolling through this menu displays an overwhelming number of options, and select **Editor font zoom out**:
    ![image](https://user-images.githubusercontent.com/35047199/107701477-d1adde80-6c6d-11eb-91f9-69c4e8b286c1.png)

Conclusion: it's possible to do what I wanted, but I only figured it out [eventually] because it was bugging me.
